etc/conf.templates/standalone/10-leveldb.conf.template

```diff
@@ -80,7 +80,32 @@ leveldb.compression.type = SNAPPY
 //
 // Maximum number of open files to use for LevelDB
 //
-#leveldb.maxopenfiles =
+#leveldb.maxopenfiles = 1000
+
+//
+// LevelDB block size (in bytes)
+//
+#leveldb.block.size = 4096
+
+//
+// LevelDB block restart interval
+//
+#leveldb.block.restart.interval = 16
+
+//
+// Flag indicating whether or not to verify block checksums
+//
+#leveldb.verify.checksums = true
+
+//
+// Flag indicating whether or not to perform paranoid checks
+//
+#leveldb.paranoid.checks = false
+
+//
+// Size of write buffer (in bytes)
+//
+#leveldb.writebuffer.size = 2097152
```

Comment:
Default value is `4 << 20 = 4194304` in `org.iq80.leveldb.Options`, see https://github.com/dain/leveldb/blob/master/leveldb-api/src/main/java/org/iq80/leveldb/Options.java#L24.